UnityProject/Assets/Scripts/NPC/AI/MobDrop.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Systems.MobAIs
+{
+	/// <summary>
+	/// Despawns mobs upon death.
+	/// Also drops an item at the same time.
+	/// </summary>
+
+	public class MobDrop : MonoBehaviour
+	{
+		private LivingHealthBehaviour mobHealth;
```

Comment:
Codacy found an issue: [Remove the field 'mobHealth' and declare it as a local variable in the relevant methods.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6375004)